### DISCUSSION TITLE
Difference between result of python api and command line usage

### DISCUSSION THREAD

**Discussion by lionsheep0724** [openai/whisper#848] [Q&A] [answered]
I saw a difference between python and command line usage.(low-high level of python, command line are all different) My audio file is about 3.5MB.
I expect that their arguments are somewhat different. Any ideas here?

**Comment by lionsheep0724** [comment]
I found more strange thing!
whisper throws diffrent results every single run, with  same .wav file.

**Comment by jzhengdj** [comment]
When doing the translate task (Japanese to English), sometimes one sentence will be repeated multiple times, causing the generated subtitle to be wrong for 10-30 seconds. Wondering how to solve this issue.

**Reply by lionsheep0724** [reply]
Many thanks to your reply!
BTW, I'm doing transcribe task, in vscode notebook. Even running for loop as below shows diffrent result as loop goes.
`for _ in range(5):
     result = model.transcribe(audio_path)
     print(result["text"])`

**Comment by glangford** [comment, marked as answer]
> I saw a difference between python and command line usage.(low-high level of python, command line are all different) My audio file is about 3.5MB. I expect that their arguments are somewhat different. Any ideas here?

The difference between command line and python execution is explained in this discussion thread
* https://github.com/openai/whisper/discussions/177

> I found more strange thing! whisper throws different results every single run, with same .wav file.

On the issue of different results across runs, see
* https://github.com/openai/whisper/discussions/81
